### PR TITLE
Dx container site root

### PR DESCRIPTION
**Pull request by jaroel**

### CODE REVIEW

File: src/plone/app/multilingual/browser/setup.py [hunk 1]
```diff
@@ -266,7 +266,7 @@ def setupLanguageSwitcher(self):
 
         if 'language-switcher' not in site.view_methods:
             methods = site.view_methods
-            site.view_methods = methods + ('language-switcher', )
+            site.view_methods = list(methods) + ['language-switcher', ]
```
Comment (mliebischer) [inline src/plone/app/multilingual/browser/setup.py]: After re-saving the FTI under ZMI -> portal_types for `Plone Site` the `site.view_methods` seems to be `tuple`. Not sure if this is a problem, but it seems related to #337.

Comment (jaroel) [reply src/plone/app/multilingual/browser/setup.py]: Thanks, that might be the cause!

File: src/plone/app/multilingual/tests/test_menu.py [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-from plone.app.multilingual.interfaces import IPloneAppMultilingualInstalled
+from plone.app.multilingual.interfaces import IPloneAppMultilingualInstalled, ITranslatable
```
Comment (jensens) [inline src/plone/app/multilingual/tests/test_menu.py]: policy is one line per import

**Review by mauritsvanrees — APPROVED** [review]
LGTM. I confirm this is needed to avoid plone.protect yelling at us directly after installing p.a.m.

I suspect this is safe to merge already, even without the related PRs in other packages. The master branch is already for Plone 6 only.
It would need another test run on Jenkins though, for only this PR.

**Comment by mauritsvanrees** [comment]
@jenkins-plone-org please run jobs

**Comment by mauritsvanrees** [comment]
As I said, I am inclined to merge this already, if green.
Or does anyone know a reason why it might not be a good idea to call `addAttributeTG` when the site root is **not** dexterity?

**Comment by mauritsvanrees** [comment]
This one is green as well now, on its own without the other dexterity site root PRs.

**Pull request merged**